cmd/managers/core/main.go

```diff
@@ -39,6 +39,7 @@ var (
 	scheme     = runtime.NewScheme()
 	setupLog   = ctrl.Log.WithName("setup")
 	syncPeriod = 10 * time.Hour
+	namespace  = os.Getenv("SYSTEM_NAMESPACE")
```

Comment:
rm

I appreciate the intent, and making the namespace dynamic is something we should presume eventually, however, it is also a much bigger effort that we should think about separately as it has downstream impact.

Reply:
Is the currently expected namespace to be always `riff-system`? This was to follow the example set forth by streaming + build to detect the config's namespace via the env defined in the yaml.

For now I've just hard coded the system namespace as "riff-system" in the config.